components/inputs/docs/input-search.md

```diff
@@ -15,7 +18,45 @@ For text searches use `<d2l-input-search>`, which wraps the native `<input type=
 </d2l-input-search>
 ```
 
-**Properties:**
+## Best Practices
+<!-- docs: start best practices -->
+<!-- docs: start dos -->
+* Specify a label for the search input and include it as part of your design. (e.g. “Search Question Library”) This is typically hidden off-screen, but may be visible when warranted. Use your discretion.
+* Add inline help to communicate when search is limited to specific facets. (e.g. “Search by question text or title”)
+* Use “Search…” (ellipsis inclusive) for placeholder text to distinguish a search input from a straight-up text input.
```

Comment:
Something about saying "straight-up text input" makes this sound too hard core 😹